packages/stark-core/src/validation/validators/is-iban/is-iban.validator.fn.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { starkIsIBAN } from "./is-iban.validator.fn";
+
+describe("Validator Function: StarkIsIBAN", () => {
+	let iban: string;
+
+	it("should fail if iban is empty", () => {
+		iban = "";
+		const result: boolean = starkIsIBAN(iban);
+		expect(result).toBe(false);
+	});
+
+	it("should fail if iban is not correct", () => {
+		iban = "BE68 5390 0754 7666";
+		const result: boolean = starkIsIBAN(iban);
+		expect(result).toBe(false);
+	});
+
+	it("should NOT fail if iban is correct", () => {
+		iban = "BE68539007547034";
```

Comment:
Why did we remove the whitespaces from here? Normally they should be fine because we pre-process the IBAN to remove them right? or am I missing something?